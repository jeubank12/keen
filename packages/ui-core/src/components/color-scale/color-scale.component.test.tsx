import React from 'react';
import { shallow } from 'enzyme';

import ColorScale from './color-scale.component';

describe('<ColorScale />', () => {
  it('should render ColorScale with gradient', () => {
    const wrapper = shallow(<ColorScale />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should render ColorScale with steps', () => {
    const wrapper = shallow(
      <ColorScale colors={['yellow', 'green', 'red']} colorMode="steps" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
