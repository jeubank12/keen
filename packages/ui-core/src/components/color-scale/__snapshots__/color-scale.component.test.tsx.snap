// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ColorScale /> should render ColorScale with discrete mode 1`] = `
<Fragment>
  <styled.div
    background="yellow"
    key="yellow"
  />
  <styled.div
    background="green"
    key="green"
  />
  <styled.div
    background="red"
    key="red"
  />
</Fragment>
`;

exports[`<ColorScale /> should render ColorScale with gradient 1`] = `
<Fragment>
  <styled.div
    endColor="#CB5623"
    startColor="#85B4C3"
  />
</Fragment>
`;
